vendor/knative.dev/operator/pkg/reconciler/common/deployments.go

```diff
@@ -52,7 +54,7 @@ func CheckDeployments(ctx context.Context, manifest *mf.Manifest, instance base.
 
 	if len(nonReadyDeployments) > 0 {
 		status.MarkDeploymentsNotReady(nonReadyDeployments)
-		return nil
+		return controller.NewRequeueAfter(5*time.Second)
```

Comment:
This can be lowered. This forces to reconcile again and make things a bit stricter.